templates/search.twig

```diff
@@ -359,15 +359,12 @@
 								{% set displayed_posts = posts %}
 							{% endif %}
 							<ul class="list-unstyled">
-								{% for post in displayed_posts %}
```

Comment:
There's much more we can remove. I limited the changes in this PR to just make it work without removing unused data.